alacritty/src/renderer/platform.rs

```diff
@@ -81,21 +81,31 @@ pub fn create_gl_context(
     gl_config: &Config,
     raw_window_handle: Option<RawWindowHandle>,
 ) -> GlutinResult<NotCurrentContext> {
-    let context_attributes = ContextAttributesBuilder::new()
-        .with_context_api(ContextApi::OpenGl(Some(Version::new(3, 3))))
-        .build(raw_window_handle);
-
-    unsafe {
-        if let Ok(gl_context) = gl_display.create_context(gl_config, &context_attributes) {
-            Ok(gl_context)
-        } else {
-            let context_attributes = ContextAttributesBuilder::new()
-                .with_profile(GlProfile::Compatibility)
-                .with_context_api(ContextApi::OpenGl(Some(Version::new(2, 1))))
-                .build(raw_window_handle);
-            gl_display.create_context(gl_config, &context_attributes)
-        }
+    let mut profiles = [
+        ContextAttributesBuilder::new()
+            .with_context_api(ContextApi::OpenGl(Some(Version::new(3, 3))))
+            .build(raw_window_handle),
+        // Try gles3 as well, since it's similar to OpenGL.
+        ContextAttributesBuilder::new()
+            .with_context_api(ContextApi::Gles(Some(Version::new(3, 0))))
+            .build(raw_window_handle),
```

Comment:
I'm surprised we need to try gles3 while gles2 just works automatically? Or is gles2 broken too?

Reply:
We don't support gles2, we need OpenGL context, but we use GLES shaders.